doc/using_kits.rst

```diff
@@ -36,7 +36,7 @@ To get started working on an existing RAMP challenge:
      .. code-block:: bash
 
         $ pip install -r requirements.txt
-        # install ramp-workflow
+        # install ramp workflow
```

Comment:
the name on pypi etc  would have a - so I would keep it as the name of the package